python/dgl/utils/__init__.py

```diff
@@ -6,3 +6,10 @@
 from .internal import *
 from .pin_memory import *
 from .shared_mem import *
+
+try:
+    from packaging import version
+except ImportError:
+    # If packaging isn't installed, try and use the vendored copy
```

Comment:
Make full use of 80 chars before change to new line. (I think it can fit in one line with just 80 chars).
End a sentence with dot.